examples/lighting-app/lighting-common/lighting-app.zap

```diff
@@ -4582,6 +4729,57 @@
               "maxInterval": 65534,
               "reportableChange": 0
             }
+          ],
+          "events": [
+            {
+              "name": "SwitchLatched",
```

Comment:
Pre-existing, but this Switch cluster on endpoint 0 does not really make much sense.  And it certainly does not have a FeatureMap that either follows the spec or matches the events enabled here....

Reply:
#24886 to follow up